easybuild/easyblocks/m/mathematica.py

```diff
@@ -112,10 +114,14 @@ def post_install_step(self):
         else:
             self.log.info("No activation key provided, so skipping activation of the installation.")
 
+        super(EB_Mathematica, self).post_install_step()
```

Comment:
I don't having this here, but you should do the symlinking you do in the easyconfig here instead (via the `symlink` function from `filetools`)?

Reply:
I decided to just add `Executables` to the PATH, since everything in `bin` is symlinking to `Executables` anyway.